### ISSUE TITLE
Javascript comment in last line doesn't beautify well

### ISSUE THREAD

**Issue by giovannibenussi** [beautify-web/js-beautify#964]
I have found that in javascript with this code the beautification doesn't work well:

``` js
res.json({  
        success: true,  
        message: []  
    })  
    // console.timeEnd("Response time")
```

However, if I delete the last line the beautification works well:

``` js
res.json({
    success: true,
    message: []
})
```

If I remove the comment, it also works:

``` js
res.json({
    success: true,
    message: []
})
console.timeEnd("Response time")
```

Regards!

**Comment by prettydiff** [comment]
https://github.com/Glavin001/atom-beautify/issues/1046

**Comment by bitwiseman** [comment]
This has the same cause as #1043 and would also benefit from the fix proposed in #1069.
(There's a an EOF token at the end of the file to which that last comment to could stick.)
